src/components/Edit_Mode_Ticket.tsx

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from "react";
+import { useEffect } from "react";
+import { EnumStatus } from "../Interface/EnumStatus";
+import { Ticket } from "./TicketItem";
+import { Form } from "react-bootstrap";
+import { Hook } from "../TicketDatabase";
+
+export function EditTicket({ ticket }: { ticket: Hook<Ticket> }): JSX.Element {
+    const initial_date = new Date();
+
+    //PART 1: HELPER functions
```

Comment:
You have a lot of functions here. Can you move them out so they are not inner functions and just pass in anything you need? For readability

Reply:
What do you mean by move them out? Put them in another file?

Reply:
No just move them outside of the `JSX.Element` function

Reply:
This request seems to now be outdated. I am not exactly sure how to move my functions out. I've never done that before and I am afraid it is going to break my code. Can I just leave as it is?

Reply:
I moved the helper functions I could move outside of my JSX.Element function. Though most functions I can't move because they use parameters of the JSX.Element function or variables in the scope of the function.

Reply:
Normally, this is resolved now. I don't think I can do better than this.